hveto/const.py

```diff
@@ -0,0 +1,33 @@
+import os
+from gwpy.time import from_gps
+
+
+def get_hvetopath(gpstime):
+
+    """ Returns the path of hveto output files
+
+    Given a gpstime, the path of the folder
+    containing hveto trigger files is returned
+
+    Parameters
+    ----------
+    gpstime : `str` or `float`
+    start time of the day for this analysis
```

Comment:
For Markdown formatting, this line should be indented by four spaces:

```suggestion
        start time of the day for this analysis
```